blockchain/proxy.go

```diff
@@ -559,6 +560,73 @@ func (ep *proxy) GetValidatorsInfoByEpoch(ctx context.Context, epoch uint32) ([]
 	return response.Data.ValidatorsInfo, nil
 }
 
+// GetESDTTokenData returns the address' fungible token data
+func (ep *proxy) GetESDTTokenData(
+	ctx context.Context,
+	address erdgoCore.AddressHandler,
+	tokenIdentifier string,
+	queryOptions api.AccountQueryOptions, // TODO: provide AccountQueryOptions on all accounts-related getters
+) (*data.ESDTFungibleTokenData, error) {
+	if check.IfNil(address) {
+		return nil, ErrNilAddress
+	}
+	if !address.IsValid() {
+		return nil, ErrInvalidAddress
+	}
+
+	endpoint := ep.endpointProvider.GetESDTTokenData(address.AddressAsBech32String(), tokenIdentifier)
+	endpoint = erdgoCore.BuildUrlWithAccountQueryOptions(endpoint, queryOptions)
```

Comment:
Consider replacing `erdgoCore` with `mxCore`.

Reply:
And `erdgoHttp` as well.

Reply:
:+1:

Reply:
good point. Will search through the entire project.